politeiad/backendv2/tstorebe/plugins/pi/pi.go

```diff
@@ -23,6 +25,26 @@ var (
 	_ plugins.PluginClient = (*piPlugin)(nil)
 )
 
+// statusData includes
+type statusData struct {
+	status pi.PropStatusT
+}
+
+// piCacheLimit limits the number of entries in the piCache context.
+const piCacheLimit = 100
```

Comment:
as we currently only store the proposal status, we might be able to increase this drastically

Reply:
i have it set to `1000` currently but i guess we could have ever higher limit as we store minimal 
data for each record ?